api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -124,8 +126,25 @@ message DownstreamTlsContext {
   OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
 }
 
+// TLS key log configuration.
+// The key log file format is "format used by NSS for its SSLKEYLOGFILE debugging output" (text taken from openssl man page)
+message TlsKeyLog {
+  // The path to save the TLS key log.
+  string tls_keylog_path = 1;
+
+  // The src IP address that will be used to filter the connection which should save the TLS key log
+  config.core.v3.Address tls_keylog_src = 2;
+
+  // The destination IP address that will be used to filter the connection which should save the TLS key log
+  config.core.v3.Address tls_keylog_dst = 3;
+
+  // Runtime flag that controls whether the key log output is enabled or not. If not specified, defaults
+  // to enabled.
+  config.core.v3.RuntimeFeatureFlag enabled = 4;
```

Comment:
If the control plane is going to configure this, why do we need this runtime flag?  Can't we just enable or disable it based on whether the configuration is present in the xDS resource?

Reply:
I agree with you. I just added it per @ggreenway's suggestion in previous review.